onnxruntime/core/providers/rocm/shared_inc/rocm_call.h

```diff
@@ -17,6 +17,7 @@ std::conditional_t<THRW, void, Status> RocmCall(
 
 #define HIP_CALL(expr) (RocmCall<hipError_t, false>((expr), #expr, "HIP", hipSuccess, "", __FILE__, __LINE__))
 #define ROCBLAS_CALL(expr) (RocmCall<rocblas_status, false>((expr), #expr, "ROCBLAS", rocblas_status_success, "", __FILE__, __LINE__))
```

Comment:
Same with here. Why not remove

Reply:
TunableOp still uses rocblas directly.  Also kernel explorer module.